lib/sort-lines.js

```diff
@@ -52,6 +52,11 @@ function lineLengthReverseCompare(a, b) {
   return a.length > b.length ? -1 : 1;
 }
 
+var intlCollator = new Intl.Collator(undefined, {numeric:true});
```

Comment:
Let's move this into the function so it doesn't get allocated unless used.

Reply:
I don't think this would be a good idea, as the function will be called for each compared pair of lines, so allocating the Collator in there would be an unnecessary overhead. Imho, a single object allocated at plugin startup is much less worse than hundreds or thousands of objects allocated on demand. If you're concerned about the variable being in the global scope, I could add a closure. 

PS: thanks for taking the time to give feedback! I'm enjoying your extension and actually looking forward to use this kind of sort myself 😉

Reply:
It's a small thing, but I mainly just don't want it to do work unless it's actually going to be used. How about you lazy load it if you're concerned about creating it every call?

```js
var intlCollator;
function naturalCompare(a, b) {
  if (!intlCollator) {
    intlCollator = new Intl.Collator(undefined, {numeric:true});
  }
  return intlCollator.compare(a, b);
}
```

Reply:
Thought this would also impact performance, but forgot about branch prediction and just-in-time optimization.